web/src/app/dialogs/new-inspection/new-inspection.component.ts

```diff
@@ -249,26 +239,92 @@ export class NewInspectionDialogComponent implements OnDestroy {
 
   public selectedStepChange(stepIndex: number) {
     if (stepIndex === NewInspectionDialogComponent.STEP_INDEX_PARAMETER_INPUT) {
-      this.dryrunRequest.next(this.currentValues.value);
+      this.dryrunRequest.next({});
     }
   }
 
   public toggleFeature(featureId: string) {
     this.featureToggleRequest.next(featureId);
   }
 
+  public run() {
+    this.hadRun.set(true);
+    of(void 0)
```

Comment:
I guess `of(void 0)` is used to trigger this flow only when the "Run" button is clicked but it might not be intuitive. 

Potential alternative:

```
(click)="run()"

private startInspectionSubject = new Subject<void>();

run() {
    this.startInspectionSubject.next();
}

ngOnInit() {
    this.startInspectionSubject.pipe(
        takeUntil(this.destroyed),
        withLatestFrom(
....
```

or `combineLatestWith` but there is a slight difference in a behavior so it depends on the expected timeline of observable emission

Reply:
Nice. I've followed your suggestion.